.github/workflows/wasmcloud-metadata.yml

```diff
@@ -0,0 +1,30 @@
+name: wasmcloud
```

Comment:
```suggestion
name: wasmcloud-metadata
```

Reply:
Yeah I was hoping that with inverse path logic that one running would cover the other requirement, but it looks like the required job isn't fulfilled still :/